jobbergate-api/jobbergate_api/tests/apps/job_submissions/test_routers.py

```diff
@@ -142,6 +163,10 @@ async def test_create_job_submission__with_client_id_in_request_body(
     assert job_submission.created_at in window
     assert job_submission.updated_at in window
 
+    assert job_submission.execution_parameters is not None
+    assert job_submission.execution_parameters.name == "job-submission-name"
+    assert job_submission.execution_parameters.comment == "I am a comment"
```

Comment:
same comment apply about checking the database apply here. If it is not clear what I mean, check this references:

[1. Check if data supposedly inserted in elasticsearch is indeed there](https://github.com/omnivector-solutions/armada-api/blob/main/tests/routers/test_agent.py#L134-L141)
[2. Check if data inserted in postgres is indeed there](https://github.com/omnivector-solutions/armada-api/blob/main/tests/routers/test_graphql.py#L276-L282)